mediorum/server/server.go

```diff
@@ -194,6 +194,7 @@ func New(config MediorumConfig) (*MediorumServer, error) {
 	routes.GET("/debug/uploads", ss.dumpUploads)
 	routes.GET("/debug/ls", ss.getLs)
 	routes.GET("/debug/peers", ss.debugPeers)
+	routes.GET("/debug/cid", ss.debugCid)
```

Comment:
this is under routes so it won't work before mediorum-first is flipped. maybe just move it to internal so the CN nginx will pick it up [here](https://github.com/AudiusProject/audius-protocol/blob/main/creator-node/nginx_conf/nginx.conf#L90)?